### PR TITLE
style: updated collection cards

### PR DESCRIPTION
**Pull request by JackShort**
* updated the collection cards
* ignore the functionality for subtraction of erc1155 its setup for the handlingErc1155 ticket

### CODE REVIEW

File: src/graphql/data/nft/Asset.ts [hunk 1]
```diff
@@ -154,7 +154,7 @@ export function useAssetsQuery(
             basePrice: ethPrice,
           }
         : undefined,
-      susFlag: asset.suspiciousFlag,
```
Comment (cbachmeier) [inline src/graphql/data/nft/Asset.ts:157]: this was an intentional refactor to not be OS specific

File: src/nft/components/collection/Card.tsx [hunk 2]
```diff
@@ -82,23 +154,35 @@ const Container = ({ asset, selected, children }: CardProps) => {
...
         transition="250"
+        cursor={asset.notForSale ? 'default' : 'pointer'}
+        onClick={(e: MouseEvent) => {
+          if (!asset.notForSale) {
+            e.preventDefault()
+            !selected ? addToBag() : removeFromBag()
+            return true
+          } else {
+            return false
+          }
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:173]: why do we need to return true/false?

File: src/nft/components/collection/Card.tsx [hunk 3]
```diff
@@ -425,100 +513,25 @@ const TertiaryInfo = ({ children }: { children: ReactNode }) => {
...
-                ? 'backgroundInteractive'
-                : selected
-                ? 'accentFailure'
-                : 'accentAction'
-              : asset.notForSale
-              ? 'backgroundModule'
-              : selected
-              ? 'accentFailureSoft'
-              : 'accentActionSoft'
-          }
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:473]: rip the flow diagrams I had to write for this logic

File: src/nft/components/collection/CollectionAsset.tsx [hunk 4]
```diff
@@ -73,39 +73,66 @@ export const CollectionAsset = ({
...
-      )}
+    <Card.Container
+      asset={asset}
+      selected={isSelected}
+      addToBag={() => {
+        addAssetsToBag([asset])
+        !bagExpanded && !isMobile && toggleBag()
+      }}
+      removeFromBag={() => {
+        removeAssetsFromBag([asset])
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionAsset.tsx:84]: add and remove to/from bag seems to be the same for assets and can be derived from hooks. Can we remove them as input parameters and implement directly in Card.Container?

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/nft/components/collection/Card.tsx (hunk 3 above)
Comment (JackShort) [reply src/nft/components/collection/Card.tsx:473]: F

File: src/nft/components/collection/Card.tsx (hunk 2 above)
Comment (JackShort) [reply src/nft/components/collection/Card.tsx:173]: this was needed for the href but no longer needed

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**